src/main/java/org/mitre/synthea/engine/ReactionProbabilities.java

```diff
@@ -0,0 +1,120 @@
+package org.mitre.synthea.engine;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.math3.distribution.EnumeratedDistribution;
+import org.apache.commons.math3.util.Pair;
+import org.mitre.synthea.world.agents.Person;
+import org.mitre.synthea.world.concepts.HealthRecord;
+
+/**
+ * ReactionProbabilities is a structure to capture the kind and severity of reaction for a
+ * given AllergyIntolerance.
+ */
+public class ReactionProbabilities implements Serializable {
+  private static final String SEVERE = "severe";
```

Comment:
It could be. For some reason, I thought this would be a better solution than having two separate enums, as I define an enum in HealthRecord that maps the severity to SNOMED codes. Not sure I can justify it.